### DISCUSSION TITLE
error with dpgen restart

### DISCUSSION THREAD

**Discussion by LZH-1996** [deepmodeling/dpgen#988] [Q&A]
after i debug the "RuntimeError: submission must have at least 1 task" error，i restart the dpgen with command line. but new error makes me submission wrong
![1](https://user-images.githubusercontent.com/46983771/194499896-642dad2a-88d5-4d8e-95da-e06971e937d1.PNG)
i changed the record dpgen to "0 2" and delete the file “01.model.devi” here is the param.json part
![2](https://user-images.githubusercontent.com/46983771/194500630-5fa6bffa-4700-42d1-bae3-44dda0ffc5bb.PNG)
![3](https://user-images.githubusercontent.com/46983771/194500641-4a6e322f-4a88-4184-8508-f7a0bc1484a9.PNG)

**Comment by HuangJiameng** [comment]
Please check if there is any empty line at the end of `record.dpgen`.

**Reply by njzjz** [reply]
The empty line can be filtered in some way.

**Reply by HuangJiameng** [reply]
> The empty line can be filtered in some way.

https://github.com/deepmodeling/dpgen/blob/devel/dpgen/generator/run.py#L3757-L3758
notice is added
